packages/webdriverio/src/scripts/isElementClickable.js

```diff
@@ -51,11 +51,12 @@ export default function isElementClickable (elem) {
                     return true
                 }
 
+                tmpElement = tmpElement.parentNode
```

Comment:
This one should not be moved, otherwise element in shadow dom won't be found

Reply:
I moved this one to fix the defect. Otherwise, tmpElement.host.parentNode is never equal to elem in the case of DocumentFragment and isClickable return false.

Reply:
While tmpElement.host = elem and isClickable return true